packages/libyaml/meta.yaml

```diff
@@ -0,0 +1,14 @@
+package:
+  name: libyaml
+  version: 0.2.1
+
+source:
+  url: https://github.com/yaml/libyaml/archive/0.2.1.zip
+
+build:
+  library: true
+  script: |
+    emcmake cmake -DCMAKE_INSTALL_PREFIX=.. .
+    emmake make -j ${PYODIDE_JOBS:-3}
+    ln -s libyaml-0.2.1/libyaml_static.a ../libyaml.a
+    ln -s libyaml-0.2.1/include ..
```

Comment:
Why do we need to create these symlinks, couldn't we just include / link to the right directory in pyyaml?

You can also use the `replace-libs` config option to rename a library when linking. See https://github.com/iodide-project/pyodide/blob/0755f191b76f2ef767f4ed3a798846ddd72717e9/packages/matplotlib/meta.yaml#L26-L27

Reply:
Actually never mind, the current approach is probably indeed better.